src/commands/image/deployImageApi/deployImage.ts

```diff
@@ -26,6 +26,9 @@ export async function deployImage(context: IActionContext & Partial<IContainerRe
         containerApp
     };
 
+    wizardContext.telemetry.properties.revisionMode = containerApp.revisionsMode;
+    wizardContext.telemetry.properties.registryDomain = context.registryDomain;
```

Comment:
I think we may want to change this to automatically be "other" in the case `context.registryDomain` is undefined.